### PR TITLE
Fix CI autorelease

### PR DESCRIPTION
**Pull request by thehatkid**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -61,79 +66,94 @@ jobs:
...
+        working-directory: ${{ runner.temp }}
+        run: |
+          mkdir release
+          cd artifacts
+          for i in */; do (cd "$i"; zip -r "../../release/${i%/}.zip" .); done
+      - name: Release ModShot
+        uses: marvinpinto/action-automatic-releases@latest
         with:
-          repo_token: "${{ secrets.GITHUB_TOKEN }}"
-          automatic_release_tag: "latest"
```
Comment (melody-rs) [inline .github/workflows/ci.yml:115]: Any reason why the auto release tag isn't set for dev?

Comment (thehatkid) [reply .github/workflows/ci.yml:115]: Now it's set to `automatic_release_tag: latest-dev`, i'm right? or I'm missing something

Comment (melody-rs) [reply .github/workflows/ci.yml:115]: Yes

Comment (melody-rs) [reply .github/workflows/ci.yml:115]: Can you please rename it back to latest

**Review by somedevfox — CHANGES_REQUESTED** [review]
Requesting fix for workflow [#19 - Release ModShot (dev)](https://github.com/Astrabit-ST/ModShot-Core/runs/6956778807?check_suite_focus=true).
Everything else is fine though!

**Comment by somedevfox** [comment]
Nevermind.. https://github.com/marvinpinto/actions/issues/61

**Review by somedevfox — APPROVED** [review]

**Comment by thehatkid** [comment]
> Nevermind..

Yes, Actions can't write repo contents such as releases and tags if from PR.

**Comment by melody-rs** [comment]
Release should not be triggering on pull request

**Comment by somedevfox** [comment]
> Release should not be triggering on pull request

Hmmm, so, the solution here would be.. splitting jobs in two separate workflow files?
One is for build, which is triggered on *everything* and one which is triggered on push-to-master?

**Comment by melody-rs** [comment]
I tried that and it didn't work very well

**Comment by thehatkid** [comment]
> I tried that and it didn't work very well

Just maybe set trigger on workflow complete?
```yml
on:
  workflow_run:
    workflows: [ModShot CI]
    types:
      - completed
```

**Comment by melody-rs** [comment]
That is basically what I did

**Comment by thehatkid** [comment]
> Release should not be triggering on pull request

Now it doesn't triggers release jobs if event is pull request.

**Comment by melody-rs** [comment]
Sounds good

**Comment by melody-rs** [comment]
Are you finished with the pr?

**Comment by thehatkid** [comment]
Yes I think, it builds and pre-releases on push and releases on tag creation

**Pull request merged**